typing/includemod.ml

```diff
@@ -245,9 +245,9 @@ let simplify_structure_coercion cc id_pos_list =
    Return the restriction that transforms a value of the smaller type
    into a value of the bigger type. *)
 
-let rec modtypes ~loc env ~mark cxt subst mty1 mty2 =
+let rec modtypes ~scope ~loc env ~mark cxt subst mty1 mty2 =
```

Comment:
The role of `scope` in those functions should be explained (so that one understands what should be passed).